### DISCUSSION TITLE
Connect with device's imei or MAC Address in iOS

### DISCUSSION THREAD

**Discussion by brown-yoo** [innoveit/react-native-ble-manager#1194] [Q&A] [answered]
I want connect to a peripheral with 'known' MAC Address. 
Because of It just already taken and stored secured server storage, I don't need to scan or find some information to connect to periphral. 

So, I can connect to peripheral without scanning it as using MAC Address in Android. 
But, In iOS, I cannot. 

When 'connect' method called, some error occurred like "Wrong UUID Format".

Couldn't I connect to device directly with MAC Address or IMEI?

Regardless of your answers, Thank you for create a good library.

**Comment by marcosinigaglia** [comment, marked as answer]
Short answer: for privacy reason you can't.
